nginx.conf

```diff
@@ -17,6 +17,7 @@ http {
   keepalive_timeout 30;
   port_in_redirect off; # Ensure that redirects don't include the internal container PORT - 8080
 
+  include mime.types;
   types {
```

Comment:
If you're moving mime type definitions to a separate file, this `types` block should be removed

Reply:
that was stupid, sorry thought i'd already nerfed that.

Reply:
done: https://github.com/alphagov/govuk-account-static-errors/commit/b619af2e6c2ba4cce5f94a1a126d5374d5441b5d